web-app/src/components/BookingModal/BookingModalForm/container.js

```diff
@@ -72,6 +75,22 @@ const Container = Wrapped =>
       });
     };
 
+    getApprovedDays = () => {
+      return getTotalDaysInEventArrayWithStatus(
+        this.props.allEvents,
+        HolidayStatus.APPROVED
+      );
+    };
+
+    getPendingDays = () => {
+      return getTotalDaysInEventArrayWithStatus(
+        this.props.allEvents,
+        HolidayStatus.PENDING
+      );
+    };
+    
+    getTotalDays = (this.props.userDetails.totalHolidays);
```

Comment:
I don't think it's good practice to assign variables like this outside of function scope in a class. The naming of this also makes me think it's a function, especially alongside `getPendingDays` etc which are actually functions.

A far cleaner way to do this would be to destructure this value from props around line `183`.